File: backend-springboot/src/main/java/com/monitor/controller/HealthController.java
package com.monitor.controller;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;

/**
 * 健康检查控制器
 */
@RestController
@RequestMapping("/health")
public class HealthController {
    
    @GetMapping
    public ResponseEntity<Map<String, Object>> health() {
        return ResponseEntity.ok(Map.of("status", "ok", "service", "monitor-backend-springboot"));
    }
}

File: backend-midway/src/controller/health.controller.ts
import { Controller, Get } from '@midwayjs/core';
import { formatDateTime } from '../utils/date.util';

@Controller('/')
export class HealthController {
  @Get('/health')
  async health() {
    return {
      status: 'ok',
      timestamp: formatDateTime(new Date()),
      service: 'monitor-backend',
    };
  }
}

